sources/api/thar-be-settings/src/service.rs

```diff
@@ -119,9 +168,11 @@ trait ServiceRestart {
     fn restart(&self) -> Result<()>;
 }
 
-impl ServiceRestart for model::Service {
+impl ServiceRestart for Service {
     fn restart(&self) -> Result<()> {
-        for restart_command in self.restart_commands.iter() {
+        let restart_commands = &self.model.restart_commands;
+        info!("restart commands {:?}", restart_commands);
```

Comment:
```suggestion
        info!("Found restart commands: {:?}", restart_commands);
```
Hmm, it's kind of unfortunate that we don't have the service name for use in messages like this.  We could change the HashMap in Services to a HashSet and store name as a field in Service instead... then we could easily use it in places like this without passing it around everywhere.

Reply:
@sanu11 raised a good point offline, which is that `Service` would have to implement `Hash` and `Eq` to be in a `HashSet`.  `model::Service` is stored in a field, and doesn't implement those traits, so it'd be a manual implementation for both... and at that point, I think it's too much code for too little benefit.